core/src/main/java/nu/yona/server/analysis/service/ActivityDTO.java

```diff
@@ -18,22 +16,25 @@
 @JsonRootName("activity")
 public class ActivityDTO
 {
-	private Date startTime;
-	private Date endTime;
+	private ZonedDateTime startTime;
+	private ZonedDateTime endTime;
 
 	@JsonCreator
-	public ActivityDTO(@JsonProperty("startTime") Date startTime, @JsonProperty("endTime") Date endTime)
+	public ActivityDTO(@JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ") @JsonProperty("startTime") ZonedDateTime startTime,
```

Comment:
Absolutely. I remembered an issue in using constants in annotations, so I didn't try it. Triggered by you, I tried it and it simply works.